app_changelog.go

```diff
@@ -0,0 +1,12 @@
+package versionbundle
+
+type AppsLog map[string][]Log
+
+type Log struct {
```

Comment:
```suggestion
type Changelog struct {
```

Reply:
It's colliding with this, which is already inside of `Bundle` https://github.com/giantswarm/versionbundle/blob/8c3879f2b9e96ab2a89b90a96b123394d08d2f6d/changelog.go#L40

Reply:
Right but the fact that we have 2 types doing the same thing is a bad design smell.

Is using the same type an option?

Or if this is only app changelogs could we call it AppCatalog?

Reply:
AppChangelog not AppCatalog. 😬

Reply:
Using the same type means we need to change the agreement around how we design apps changelog on `releases`. 
```
- version: 0.11.2
  description: Remove CPU Limits
  type: Changed
  urls:
    - https://github.com/giantswarm/coredns/pull/335
- version: 0.11.1
  ...
```
For example, `Changelog` in `Bundle` does not include `version` which is crucial to app changelog. So I would like to go with AppChangelog, WDYT?

Reply:
Yes this is because how we store changelogs in operators and releases is different.

```go
		Changelogs: []versionbundle.Changelog{
			{
				Component:   "net-exporter",
				Description: "Update to 1.6.0.",
				Kind:        versionbundle.KindChanged,
				URLs: []string{
					"https://github.com/giantswarm/net-exporter/releases/tag/v1.6.0",
				},
			},
		},
```

But they are both changelogs and we can have optional fields. So can we add Version and ComponentVersion to the existing struct?

I'd like us to try that and we can get feedback from other teams. Rather than adding another struct when we don't need one.

Reply:
> So can we add Version and ComponentVersion to the existing struct? 
Sure we can try this.

Reply:
Since the original `kind` type in `versionbundle` is not following capital alphabet in words, like `KindChanged: changed`, I need to change either app changelog to follow `versionbundle` or change versionbundle and update vendoring all operators :(. 

@rossf7 I think former makes sense. WDYT?

Reply:
We agreed upon https://github.com/giantswarm/giantswarm/pull/8705, so going with the former.